### PR TITLE
Ava UI: Various Fixes

### PR DESCRIPTION
**Pull request by IsaacMarovitz**
- Fixed SaveManager bug in #4166 
- Allow save size to be formatted as KB and GB
- Move SearchBox fix in #4223 to `Styles`

### CODE REVIEW

File: Ryujinx.Ava/UI/Models/SaveModel.cs [hunk 1]
```diff
@@ -41,11 +36,27 @@ public long Size
...
+        private string GetSizeString()
+        {
+            const int scale = 1024;
+            string[] orders = { "GB", "MB", "KB" };
+            long max = (long)Math.Pow(scale, orders.Length);
+
+            foreach (string order in orders)
+            {
+                if ( Size > max )
+                    return $"{decimal.Divide(Size, max):##.##} {order}";
```
Comment (AcK77) [inline Ryujinx.Ava/UI/Models/SaveModel.cs:50]:
```suggestion
                if ( Size > max )
                {
                    return $"{decimal.Divide(Size, max):##.##} {order}";
                }
```

File: Ryujinx.Ava/UI/Models/SaveModel.cs [hunk 2]
```diff
@@ -41,11 +36,27 @@ public long Size
...
         public bool SizeAvailable { get; set; }
 
-        public string SizeString => $"{((float)_size * 0.000000954):0.###}MB";
+        public string SizeString => GetSizeString();
 
-        public SaveModel(SaveDataInfo info, HorizonClient horizonClient, VirtualFileSystem virtualFileSystem)
+        private string GetSizeString()
+        {
+            const int scale = 1024;
+            string[] orders = { "GB", "MB", "KB" };
```
Comment (AcK77) [inline Ryujinx.Ava/UI/Models/SaveModel.cs:44]:
```suggestion
            string[] orders = { "GiB", "MiB", "KiB" };
```
In a way to follow what have been done in https://github.com/Ryujinx/Ryujinx/pull/3716

File: Ryujinx.Ava/UI/Models/SaveModel.cs [hunk 3]
```diff
@@ -41,11 +36,27 @@ public long Size
...
+
+            foreach (string order in orders)
+            {
+                if ( Size > max )
+                    return $"{decimal.Divide(Size, max):##.##} {order}";
+
+                max /= scale;
+            }
+
+            return "0 KB";
```
Comment (AcK77) [inline Ryujinx.Ava/UI/Models/SaveModel.cs:55]:
```suggestion
            return "0 KiB";
```
In a way to follow what have been done in https://github.com/Ryujinx/Ryujinx/pull/3716

**Review by AcK77 — CHANGES_REQUESTED** [review]

**Review by AcK77 — APPROVED** [review]
Lgtm! Thanks!

**Review by marysaka — APPROVED** [review]

**Pull request merged**